mappings/net/minecraft/client/search/PlainTextSearch.mapping

```diff
@@ -1,8 +1,8 @@
 CLASS net/minecraft/unmapped/C_vsqjjavd net/minecraft/client/search/PlainTextSearch
-	FIELD f_betspott plainTextSearchTree Lnet/minecraft/unmapped/C_kajwzxdj;
+	FIELD f_betspott plainTextSearcher Lnet/minecraft/unmapped/C_kajwzxdj;
 	FIELD f_kugqkwbk filler Ljava/util/function/Function;
-	FIELD f_orttexni values Ljava/util/List;
+	FIELD f_orttexni searchables Ljava/util/List;
 	METHOD <init> (Ljava/util/function/Function;Ljava/util/function/Function;Ljava/util/List;)V
 		ARG 1 filler
-		ARG 2 ids
-		ARG 3 values
+		ARG 2 identifiers
```

Comment:
`ids` is a [conventional abbreviation](https://github.com/QuiltMC/quilt-mappings/blob/1.20-pre6/CONVENTIONS.md#abbreviations) for `identifiers`